Sources/SwifterMedia.swift

```diff
@@ -22,16 +22,18 @@ public enum MediaCategory: String {
 }
 
 public extension Swifter {
-
-    internal func prepareUpload(data: Data, type: MediaType, category: MediaCategory, success: JSONSuccessHandler? = nil, failure: FailureHandler? = nil) {
+    
+    internal func prepareUpload(data: Data, type: MediaType, category: MediaCategory? = nil, success: JSONSuccessHandler? = nil, failure: FailureHandler? = nil) {
         let path = "media/upload.json"
-        let parameters: [String : Any] = ["command": "INIT",
+        var parameters: [String : Any] = ["command": "INIT",
                                           "total_bytes": data.count,
-                                          "media_type": type.rawValue,
-                                          "media_category": category.rawValue]
+                                          "media_type": type.rawValue]
+        if let cat = category {
+            parameters["media_category"] = cat
+        }
         self.postJSON(path: path, baseURL: .upload, parameters: parameters, success: success, failure: failure)
     }
-
+    
```

Comment:
Please remove all the empty spaces you've added here. (Set Xcode to trim whitespace lines)